src/Orckestra.Composer.Cart/ViewModels/Order/OrderDetailInfoViewModel.cs

```diff
@@ -79,5 +79,14 @@ public sealed class OrderDetailInfoViewModel : BaseViewModel
         /// The source of the order.
         /// </value>
         public string Source { get; set; }
+
+
+        /// <summary>
+        /// Gets or sets the Is Order In Editable Shipment Status of Order
+        /// </summary>
+        /// <value>
+        /// The source of the order.
+        /// </value>
+        public bool IsOrderInEditableShipmentStatus { get; set; }
```

Comment:
rename this to IsOrderEditable - as to calculate this value client can use different logic